server/services/unusedDeductionsService.ts

```diff
@@ -82,7 +89,11 @@ export default class UnusedDeductionsService {
 
       return ['UNKNOWN', adjustments]
     } catch {
-      return ['UNKNOWN', adjustments]
+      if (this.getTotalUnusedRemand(adjustments) > 0) {
+        return ['CRDS_CALL_FAILED', adjustments]
```

Comment:
This isn't quite right. This banner is going to replace the `UNSUPPORTED` banner, so don't need to add any more `UnusedDeductionMessageType`s